src/data/keys/countryCodes.js

```diff
@@ -0,0 +1,260 @@
+export const CC_AC = 'AC';
```

Comment:
Goed dat je dit zo opgezet hebt. Ik zit echter in twee strijd;

Ik juig het idee toe om dit in variabelen te zetten, maar ik vind het niet echt leesbaarder.

Als ik nu die hele lijst met landen zie, weet ik niet aan de hand van variabele naam om welk land het specifiek gaat.

Mocht je het met mij eens zijn; Mijn oordeel voor nu is: lekker laten staan; maar wellicht een goeie om te onthouden en evt te refactoren.